### PR TITLE
feat: data cleaning for data science onramp

### PR DESCRIPTION
**Pull request by TusharK54**
Adding code for data cleaning section of the Data Science Onramp tutorial.

### CODE REVIEW

**Comment by leahecole** [comment]
Commits from vuppalli and Symmetries were made while they were actively interns at Google and while they had a signed CLA

File: data-science-onramp/data-cleaning/requirements-test.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pytest==6.0.0
+pandas==1.1.0
+google-cloud-bigquery==1.26.1
+pyarrow==1.0.0
```
Comment (bradmiro) [inline data-science-onramp/data-cleaning/requirements-test.txt:4]: What is `pyarrow` being used for?

Comment (TusharK54) [reply data-science-onramp/data-cleaning/requirements-test.txt:4]: It is required to load a table from BigQuery. I get the following errors when I run without it:
```
ImportError: Unable to find a usable engine; tried using: 'pyarrow', 'fastparquet'.
A suitable version of pyarrow or fastparquet is required for parquet support.
```
```
PyarrowMissingWarning: Loading dataframe data without pyarrow installed is deprecated and will become unsupported in the future. Please install the pyarrow package.
```

Comment (bradmiro) [reply data-science-onramp/data-cleaning/requirements-test.txt:4]: Sounds good, lgtm

**Comment by leahecole** [comment]
CLA Bot can cool it these were interns who are just done with their internship - all is well :)

**Pull request merged**